.github/workflows/ci.yaml

```
@@ -0,0 +1,23 @@
+name: Dart CI
+
+on:
+  push:
+    branches: [ master ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)